backend/apps/notifications/views.py

```
@@ -5,6 +5,26 @@
 from .models import Notification, PushSubscription
 from .serializers import NotificationSerializer, PushSubscriptionSerializer
 
+from .models import NotificationPreference
+
+class NotificationPrefsView(APIView):
+    permission_classes = [IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)